src/common/src/DocumentViewer/DocumentRenderer.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Alert, Link, Paragraph } from '@amsterdam/asc-ui';
+import { DocumentRendererStyle } from './DocumentRendererStyles';
+
+type Props = {
+	uri: string;
+	contentType: string;
+};
+
+export const imageContentTypes: string[] = [
```

Comment:
Have you considered making this a TS enum? In that way you could have type safety on the passed strings.

Reply:
Done